omod/src/main/java/org/openmrs/module/bacteriology/web/v1_0/resource/SpecimenResource.java

```diff
@@ -38,7 +38,7 @@
 
 import java.util.*;
 
-@Resource(name = RestConstants.VERSION_1 + "/specimen", supportedClass = Specimen.class, supportedOpenmrsVersions = {"1.10.*", "1.11.*", "1.12.*", "2.0.*", "2.1.*"})
+@Resource(name = RestConstants.VERSION_1 + "/specimen", supportedClass = Specimen.class, supportedOpenmrsVersions = {"1.10.*", "1.11.*", "1.12.*", "2.0.*", "2.1.*", "2.2.*", "2.3.*", "2.4.*"})
```

Comment:
Our current recommendation for this is to do it as: `1.10.* - 9.*`

Reply:
That way, we do not have the maintenance burden of adjusting it for each platform version that is released. We do the changes only when a particular platform version introduces some incompatibility.

Reply:
@dkayiwa Addressed feedback comments